migrations/20241107115400_count_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE envelopes
+ADD COLUMN count INTEGER DEFAULT 1 NOT NULL;
```

Comment:
We need to track envelope counts per row so we can count how many envelopes are in the db at startup.